bandit/blacklists/calls.py

```diff
@@ -685,6 +698,17 @@ def gen_blacklist():
         )
     )
 
+    sets.append(
+        utils.build_conf_dict(
+            "pytorch_load_save",
+            "B704",
+            issue.Cwe.DESERIALIZATION_OF_UNTRUSTED_DATA,
```

Comment:
This seems to mismatch what's in the plugin. Maybe I'm misremembering how these all work

Reply:
Happy to change it, I selected DESERIALIZATION_OF_UNTRUSTED_DATA as its the same (I think they abstract / wrap pickle).

Reply:
I'm fine with either. Just pointing out that the plugin registration and this seems to disagree